app/views/candidate_mailer/duplicate_match_email.text.erb

```diff
@@ -1,13 +1,7 @@
 Dear <%= @application_form.first_name %>
 
-You’ve created more than one account on Apply for teacher training.
+You created more than one account to apply for teacher training. Both of your accounts have been locked.
```

Comment:
We try to avoid capitalising the service name where possible (as users don't usually think of it as a Thing, but rather a bit of GOV.UK).

"Your accounts have been locked." sounds sensible to me though, to cover any cases where 3 accounts have been locked? (Unless we could add logic for this?)

Reply:
That is a fair point - technically it is possible if someone creates 3+ accounts within an hour (we changed the clock job to check more frequently than daily). I will amend this.

Reply:
Pushed. Resolving this comment.